distribution_circuits_website_sale/controllers/main.py

```diff
@@ -30,8 +30,7 @@ def _get_search_domain(self, search, category, attrib_values):
         '/shop',
         '/shop/page/<int:page>',
         '/shop/category/<model("product.public.category"):category>',
-        '/shop/category/<model("product.public.category"):category'
-        '>/page/<int:page>',  # Continue previous line
+        '/shop/category/<model("product.public.category"):category>/page/<int:page>',  # noqa
```

Comment:
I was not aware about `#noqa` thanks. :)

Reply:
Don't abuse it ;-)